docs/source/basics/packing.rst

```diff
@@ -0,0 +1,52 @@
+.. _packing_usage_label:
```

Comment:
you snuck this in here you sneaky lil man

i love it

Reply:
heh nice

Reply:
ok I just now realized the joke